staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -127,3 +135,24 @@ func TestParserV1beta1(t *testing.T) {
 		t.Errorf("Unexpected policy! Diff:\n%s", diff.ObjectDiff(policy, expectedPolicy))
 	}
 }
+
+func TestPolicyCntCheck(t *testing.T) {
+
+	test := func(policyFileName string) {
+		// Create an illegal policy file.
+	        f, err := ioutil.TempFile("", "illegalPolicy.yaml")
+		require.NoError(t, err)
+		defer os.Remove(f.Name())
+
+		_, err = f.WriteString(policyFileName)
+		require.NoError(t, err)
+		require.NoError(t, f.Close())
+
+		_, err = LoadPolicyFromFile(f.Name())
+		assert.EqualError(t, err, fmt.Sprintf("loaded illegal policy with 0 rules from file %s", f.Name()))
```

Comment:
I don't think copy-pasting the error to compare to is a good idea. If you change the message, you will have to change this place also. IMHO, a presence check will be enough

Reply:
Done, plz check ^_^